packages/odyssey-react-mui/src/Callout.tsx

```diff
@@ -40,20 +52,49 @@ export type CalloutProps = {
    * Determine the color and icon of the Callout
    */
   severity: (typeof calloutSeverityValues)[number];
+  /**
+   * The text content of the Callout
+   */
+  text?: string;
   /**
    * The title of the Callout
    */
   title?: string;
-} & SeleniumProps;
+} & (
+  | {
+      text?: string;
+      children: ReactNode;
+    }
+  | {
+      text: string;
+      children?: ReactNode;
+    }
+) &
+  SeleniumProps;
 
-const Callout = ({ children, role, severity, testId, title }: CalloutProps) => {
+const Callout = ({
+  children,
+  linkText,
+  linkUrl,
+  role,
+  severity,
+  testId,
+  text,
+  title,
+}: CalloutProps) => {
   const { t } = useTranslation();
 
   return (
     <Alert data-se={testId} role={role} severity={severity} variant="callout">
       <ScreenReaderText>{t(`severity.${severity}`)}</ScreenReaderText>
       {title && <AlertTitle>{title}</AlertTitle>}
-      <Box component="div">{children}</Box>
+      {children && <Box component="div">{children}</Box>}
+      {text && <Paragraph>{text}</Paragraph>}
+      {linkUrl && (
+        <Link href={linkUrl} variant="monochrome">
+          {linkText}
```

Comment:
Just checking.. is there a default value for `linkText`? I didn't see one as part of a change but it may already exist in the code.

Reply:
No, the consumer has to specify it.